gunicorn/arbiter.py

```diff
@@ -522,11 +525,6 @@ def spawn_worker(self):
             sys.exit(-1)
         finally:
             self.log.info("Worker exiting (pid: %s)", worker_pid)
-            try:
-                worker.tmp.close()
-                self.cfg.worker_exit(self, worker)
-            except:
-                pass
```

Comment:
Removed because this portion of `spawn_worker` is executed on the worker itself, and we let the parent handle its exit in `reap_workers`.